CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## [3.2.1] - Aug 15, 2024
+
+### Bug fixes 
+- `clientReady` is true even though internal client promise returns `success == false` bug fix
+- `useDecision` hook set the update listener on overy render bug fix
+- `setForcedDecision` does not reflect the changes in optmizely instance and `useDecision` hook bug fix
```

Comment:
Could you add the references to the PRs that solve these?  Check-out the examples from earlier.

Reply:
My bad! Thanks!